packages/fail-logs/hooks/inject-failure-summary.py

```
@@ -329,7 +331,24 @@ def main() -> None:
     logs_dir = (
         logs_dir_value if isinstance(logs_dir_value, str) and logs_dir_value else DEFAULT_LOGS_DIR
     )
-    log_path = _resolve_log_path(project_dir, logs_dir)
+
+    # log_root の解決（git サブプロセス起動）は enabled / summary.enabled 判定の
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)